apps/console/src/features/users/pages/users.tsx

```diff
@@ -637,7 +632,7 @@ const UsersPage: FunctionComponent<UsersPageInterface> = (
      * Handles the click event of the create new user button.
      */
     const handleAddNewUserWizardClick = (): void => {
-
+        setConnecterConfigLoading(true);
```

Comment:
Instead of adding new state variables to handle loading states, let's replace `getConnectorCategory` with `useGovernenceConnectorCategories` [1].

[1] https://github.com/wso2/identity-apps/blob/2b7fc0033fe87e18174e20e5acb296a824d63b67/apps/console/src/features/server-configurations/api/governance-connectors.ts#L50

Reply:
`useGovernanceConnectorCategories` hook doesn't provide the required response.